java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -119,6 +120,11 @@ public void unlink_multiple_keys(BaseClient client) {
 
         Long unlinkedKeysNum = client.unlink(new String[] {key1, key2, key3}).get();
         assertEquals(3L, unlinkedKeysNum);
+
+        // command can handle cross-slot case
```

Comment:
can't we just put all the cross-slot commands into a single unit test in java/integTest/src/test/java/glide/cluster/ClusterClientTests.java? 

Same for single-node commands?

Reply:
will do once we merge all these commands to avoid extra mess